Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -0,0 +1,155 @@
+import KsApi
```

Comment:
Is `KsApi` import needed? (Sorry reviewing this on my phone)

Reply:
No worries, and eventually if we plan to configure w/ `Reward`. I thought about doing that w/ this PR but decided not to and did not remove it.